dps-gradle-spring-boot-suppressions.xml

```diff
@@ -292,4 +292,32 @@
     <packageUrl regex="true">^pkg:maven/com\.nimbusds/lang\-tag@.*$</packageUrl>
     <cve>CVE-2020-23171</cve>
   </suppress>
+  <suppress>
```

Comment:
Can just .gitignore this whole file and delete it from the repo - it gets autogenerated during the build